client/src/components/SearchPage/SearchField/SearchField.js

```diff
@@ -1,17 +1,19 @@
-import React, { Component } from "react"
-import classes from "./SearchField.module.css"
-import Logo from "../../assets/logo.png"
-import Search from "../../assets/search.png"
-import data from "./data"
+import React, { Component } from 'react'
+import classes from './SearchField.module.css'
+import Logo from '../../assets/logo.png'
+import Search from '../../assets/search.png'
+import data from './data'
+import { withRouter } from 'react-router-dom'
 
 class SearchField extends Component {
     constructor(props) {
         super(props)
         this.state = {
             words: [],
-            userInput: "",
+            userInput: '',
             matches: [],
             rowHighlighted: -1,
+            query: '',
         }
     }
 
```

Comment:
Presently, if a user wants to search for "react" pathways and enters just "rea" in the search field, no results are displayed. All the pathways whose names include the string entered by the user should be displayed on the search result page. Also, when the user enters any string, the options displayed to the user with matching names are random words exported from data.js file. Remove that file and show the actual pathway names as the options to the user.